HomeAssistant/Authentication/AuthenticationController.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  AuthenticationController.swift
+//  HomeAssistant
+//
+//  Created by Stephan Vanterpool on 8/11/18.
+//  Copyright © 2018 Robbie Trencheny. All rights reserved.
+//
+
+import Foundation
+import PromiseKit
+import SafariServices
+let kAuthenticationPath = "/auth/authorize?response_type=code&client_id=https%3A//blackgold9.github.io"
```

Comment:
Let's change this `client_id` to be like` `https://home-assistant.io/ios`?

Reply:
We can change it, but that address needs to host a page with a specific tag in it's header. Don't know how to make the current home assistant webpage do tat.